README.md

```diff
@@ -149,12 +153,24 @@ void error(String message);
 void end();
 ```
 
+`start()` records the active PurePath node on the current Java thread
+as parent (if any; whether created by another Tracer or the OneAgent), creates a new PurePath node
+and sets the new one as the currently active one. The OneAgent also requires that a child node ends
+before all parent nodes (Stated another way, tracers on the same thread must be ended in the opposite
```

Comment:
"(in other words, ...)" - note the change of case

Reply:
```suggestion
before all parent nodes (in other words, tracers on the same thread must be ended in the opposite
```

Is this your suggestion? I agree that it sounds a bit better and the casing at the moment is also wrong. If I remember to do so, I will submit PRs against our internal repository and then GitHub, but I think it is not a deal-breaker to leave it as-is.